operator/main.go

```diff
@@ -119,7 +119,7 @@ func main() {
 	}
 	provider := metric.NewMeterProvider(metric.WithReader(exporter))
 	meter := provider.Meter("keptn/task")
-	deploymentCount, err := meter.SyncInt64().Counter("keptn.deployment.count", instrument.WithDescription("a simple counter for Keptn Deployments"))
+	deploymentGauge, err := meter.AsyncInt64().Gauge("keptn.deployment.count", instrument.WithDescription("a simple counter for Keptn Deployments"))
```

Comment:
Gauges are non additive which is in contrast with the intent of `keptn.deployment.count` to be additive. Furthermore, the counting is not an observation but a synchronous read. 

https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md#asynchronous-gauge

Reply:
This is also valid for the remaining counters in this file